packages/contracts-bedrock/contracts/test/GasPriceOracle.t.sol

```diff
@@ -81,7 +82,8 @@ contract GasPriceOracle_Test is CommonTest {
 
     function test_l1BaseFee() external {
         uint256 l1BaseFee = gasOracle.l1BaseFee();
-        assertEq(l1BaseFee, 100);
+        uint256 expect = Lib_FeeSmoothing.rollingAverage(0, 100);
```

Comment:
The value here is `30` which I believe is realistic if 70% weighted towards prev value and 30% weighted towards next value. We need unit tests for the `rollingAverage` function, not 100% sure of the best way to go about it